src/expressions/operator-expr.md

```diff
@@ -101,14 +101,14 @@ struct Packed {
 
 let packed = Packed { f1: 1, f2: 2 };
 // `&packed.f2` would create an unaligned reference, and thus be Undefined Behavior!
-let raw_f2 = ptr::addr_of!(packed.f2);
+let raw_f2 = &raw const packed.f2;
```

Comment:
I assume this will fail in CI since the operator is not stable yet. Not sure how you usually stage this.

Reply:
Yea, CI will just be red until the stabilization PR hits nightly, after which point we can merge this.